INSTALL.md

```diff
@@ -245,8 +245,8 @@ Then follow step 4 in [Instructions for all OSes](#instructions-for-all-oses) be
      type `opam switch list-available`
      to see what versions are available and then `opam switch <version-number>`.
 
-   - Afterwards you can also install the `depext` package,
-     to be able to install some binary dependencies below more easily.
+   - Afterwards you can also install the `depext` package if you are on OPAM version lower then 2.1,
+     to be able to install some binary dependencies below more easily. Version of OPAM after 2.1 has depext handling backed in.
```

Comment:
s/backed/baked/

Reply:
Done. Can I ask you try `suggestions` next time? I think that feature nice exact for this kind of issues which crops up.